docs/source/schema-design/federated-schemas/reference/directives.mdx

```diff
@@ -1164,6 +1164,146 @@ The default value is `true`.
 </tbody>
 </table>
 
+## Managing cache invalidation
+
+### `@cacheTag`
+
+<MinVersionBadge version="Federation v2.12" />
+
+```graphql
+directive @cacheTag(format: String!) repeatable on FIELD_DEFINITION | OBJECT
+```
+
+Assigns cache tags to cached data in the Apollo Router for [active cache invalidation](/graphos/routing/performance/caching/response-caching/invalidation#active-invalidation). Use cache tags to remove specific cached entries on demand when data changes, rather than waiting for time-to-live (TTL) expiration.
+
+```graphql {7-8}
+extend schema
+  @link(url: "https://specs.apollo.dev/federation/v2.12",
+        import: ["@key", "@cacheTag"])
+
+type Query {
+  users: [User!]! @cacheTag(format: "users-list")
+  user(id: ID!): User @cacheTag(format: "user-{$args.id}")
+}
+
+type User @key(fields: "id") @cacheTag(format: "user-{$key.id}") {
+  id: ID!
+  name: String!
+}
+```
+
+<Note>
+
+The `@cacheTag` directive is defined in your subgraph schema, but it's only interpreted by the router—your subgraph doesn't use it at all. This differs from `@cacheControl`, which is both defined and interpreted at the subgraph level.
+
+</Note>
+
+You can combine `@cacheTag` with `@cacheControl` to set reasonable TTLs as a safety net while maintaining the ability to invalidate data immediately when needed.
+
+#### Arguments
+
+<table class="field-table">
+  <thead>
+    <tr>
+      <th>Name /<br/>Type</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+
+<tbody>
+
+<tr class="required">
+<td>
+
+##### `format`
+
+`String!`
+
+</td>
+<td>
+
+**Required.** A string template that defines the cache tag. Can include interpolated variables:
+- For root fields: `{$args.fieldName}` to interpolate field arguments
+- For entities: `{$key.fieldName}` to interpolate entity key fields
+
+The `format` must always generate a valid string (not an object).
+
+Examples:
+
+* `"users-list"` (static tag)
+* `"user-{$args.id}"` (root field with argument)
+* `"product-{$key.upc}"` (entity with key field)
+
+</td>
+</tr>
+
+</tbody>
+</table>
+
+#### Usage
+
+Apply `@cacheTag` to root query fields or entity types (types with `@key` where `resolvable` is unset or `true`).
```

Comment:
```suggestion
Apply `@cacheTag` to root query fields or entity types.
```
The stuff in parentheses looks like it's trying to describe entity types. I don't think we need to do that here.